calico-cloud/networking/egress/egress-gateway-azure.mdx

```diff
@@ -0,0 +1,678 @@
+- [(Optional) Enable ECMP load balancing](#optionally-enable-ecmp-load-balancing)
+- [Verify the feature operation](#verify-the-feature-operation)
+- [Control the use of egress gateways](#control-the-use-of-egress-gateways)
+- [Policy enforcement for flows via an egress gateway](#policy-enforcement-for-flows-via-an-egress-gateway)
+
+### Choose route reflectors
+
+It is possible to establish BGP connections between all {{prodname}} nodes and Azure Route Servers, but
+to avoid hitting Azure Route Server peers [limit](https://learn.microsoft.com/en-us/azure/route-server/overview#route-server-limits),
+it is better to select some nodes as route reflectors and set up BGP connections between those and Azure Route Server.
+The number of route reflectors depends on the cluster size, but it is recommended to have at least 3 at all times,
+and Azure Route Servers supports up to 8 peers in a VNet.
+
+### Create Azure Route Server
+
+Deploy Azure [Route Server](https://learn.microsoft.com/en-us/azure/route-server/) in the VNet with egress gateways,
+and add the chosen route reflectors as peers to it.
+
+:::note
+
+- The BGP connections between Calico route reflectors and Azure Route Servers are critical for the functionality
+of egress gateways. It is important to maintain route reflectors with care, and to make sure there are always
+enough healthy route reflectors.
+- If possible, assign a static address to the route reflectors so after reboots the same address is kept.
+- In AKS, it is [recommended](https://learn.microsoft.com/en-us/azure/aks/use-system-pools?tabs=azure-cli#system-and-user-node-pools)
+to run applications in user node pools and leave system node pools for running critical system pods.
+The nodes in system node pools are perfect Route Reflector candidates.
+
+:::
+
+### Disable the default BGP node-to-node mesh
+
+The default {{prodname}} **node-to-node BGP mesh** may be turned off to enable other BGP topologies.
+To do this, modify the default **BGP configuration** resource.
+
+Run the following command to disable the BGP full-mesh:
+
+```batch
+kubectl apply -f - <<EOF
+apiVersion: projectcalico.org/v3
+kind: BGPConfiguration
+metadata:
+  name: default
+spec:
+  nodeToNodeMeshEnabled: false
+EOF
+```
+
+:::note
+
+Disabling the node-to-node mesh will break pod networking until/unless you configure replacement BGP peerings using BGPPeer resources.
+You may configure the BGPPeer resources before disabling the node-to-node mesh to avoid pod networking breakage.
+
+:::
+
+### Enable BGP
+
+Usually, in the default installation of {{prodname}} in a public cloud, BGP is disabled as it is not used.
+However, running egress gateways in Azure relies on BGP connections between {{prodname}} and Azure
+Route Servers. To enable BGP routing protocol, set `bgp` field to `Enabled` in the default installation:
+
+```batch
+kubectl patch installation default --type='merge' -p '{"spec": {"calicoNetwork": {"bgp": "Enabled"}}}'
```

Comment:
This won't work for AKS clusters that had AKS managed Calico and then were upgraded to Enterprise or Calico Cloud. For those installations the name of the installation resource would be `overlay`.
I don't know if in general the docs handle adjusting the name for the installation resource for clusters that use the `overlay` resource.